src/main/java/com/greenfiling/smclient/model/exchange/SupplierCostFilter.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright 2023 Green Filing, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.greenfiling.smclient.model.exchange;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+
+import com.greenfiling.smclient.model.internal.FilterBase;
+
+public class SupplierCostFilter extends FilterBase {
+
+  public static String JOB_TYPE_SOP = "1";
```

Comment:
I see your change for this.  Seeing the change drew my attention to the fact that jobTypeId and slaId are modeled as Strings instead of Integers. Is there a reason for this? They're integers in the object returned from SM.

One possible reason is because we're going to filter on them and addListFilter and FilterPair expect strings?  Maybe we could have addListFilter accept an ArrayList of Objects and call .toString() on them?  I think that would work and if it does I would prefer that over modeling ints as Strings

Reply:
Originally I made them Strings to be less strict with the data type in the event SM introduces non-integer codes for the fields. I agree that it may not be a good decision.